css/properties/border-image-slice.json

```diff
@@ -0,0 +1,64 @@
+{
+  "css": {
+    "properties": {
+      "border-image-slice": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/border-image-slice",
+          "support": {
+            "webview_android": {
+              "prefix": "-webkit-",
+              "version_added": "4.1"
+            },
+            "chrome": {
+              "version_added": "15"
+            },
+            "chrome_android": {
+              "version_added": null
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "15",
+              "notes": [
```

Comment:
For these notes and the equivalent notes for Android, I've broken them into separate footnotes (unlike the original table, which had them as one), since they represented different bugs and behaviors. If this was not the right thing to do, let me know and I'll squash them back into one.

Reply:
Thanks for this. So much better!